common/src/leap/soledad/common/couch.py

```diff
@@ -476,9 +434,14 @@ def __init__(self, url, dbname, replica_uid=None, ensure_ddocs=True):
             self._set_replica_uid(replica_uid)
         if ensure_ddocs:
             self.ensure_ddocs_on_db()
-        # initialize a thread pool for parallelizing get_docs()
-        self._sem_pool = threading.BoundedSemaphore(
-            value=self.MAX_GET_DOCS_THREADS)
+        self._cache = None
```

Comment:
Good catch! Will add a init_caching to this instead of using private _cache outside of the class.
I needed to do this because soledad.common doesn't have caching and shouldn't as beaker is only for soledad.server